libs/ledger-live-common/src/apps/support.ts

```diff
@@ -37,8 +37,11 @@ const appVersionsRequired = {
   Solana: ">= 1.2.0",
   Celo: ">= 1.1.8",
   "Cardano ADA": ">= 4.1.0",
-  Zcash: "> 2.0.6",
+  Bitcoin: ">= 2.1.0",
```

Comment:
good point.
but I think we don't have to be worried about that. because since bitcoin (and other bitcoin family apps) 2.1.0
none of them depends on bitcoin nano app at this moment. (refer to https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/apps/polyfill.ts#L73-L78)
I've just now put all the bitcoin family apps in the appVersionsRequired list. So users should not have any problem about it.

Reply:
Agree with Huiqi, not dependency anymore between bitcoin apps.

However for some coins in the list, this may force a firmware update? I don't remember why but this seemed to be an issue in the past, maybe for users on LLM?